bcda/cclf/cclf.go

```diff
@@ -419,7 +419,7 @@ func sortCCLFArchives(cclfMap *map[string]map[int][]*cclfFileMetadata, skipped *
 			log.Errorf("Unknown file found: %s", metadata)
 			*skipped = *skipped + 1
 
-			deleteThreshold := time.Hour * time.Duration(utils.GetEnvInt("ARCHIVE_THRESHOLD_HR", 24))
+			deleteThreshold := time.Hour * time.Duration(utils.GetEnvInt("BCDA_ETL_FILE_ARCHIVE_THRESHOLD_HR", 72))
```

Comment:
I don't see this env var set elsewhere.  Will we be relying on the default, or setting the env var separately?

Reply:
Yeah, i assume we'll be relying on the default. I think that's standard practice at this point :)

Reply:
@dhgreene Yes, standard practice has been to just rely on the default and only set/create an env var if we plan to override the default.